jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -249,6 +257,45 @@ node {
         currentBuild.description = "latest: ${latestRelease}\ncurrent: ${componentRelease}"
     }
 
+    stage('Fetch Component Product Version') {
+        // Only set the product version as the latest release if the release tag var is not available
+        if (!releaseTagVar) {
+          productVersion = latestRelease
+        }
+        // Ensures that old jobs will still work with old parameter
+        // Can be removed after all service discovery jobs have been updated
+        else if (params.manifestVar) {
+          productVersion = latestRelease
+        } else {
+          if (!productVersionLocation || !productVersionIdentifier) {
+            println "[ERROR] The product version location and/or identifier was not defined"
```

Comment:
Should this break here or something? I'm assuming if both of these are missing then it can't really do anything?